services/blockchain-indexer/methods/indexer.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+	triggerAccountUpdates,
+	indexAllDelegateAccounts,
+	cacheLegacyAccountInfo,
+	indexGenesisAccounts,
+	indexNewBlock,
+	indexMissingBlocks,
+	getMissingBlocksListByRange,
+} = require('./controllers/indexer');
+
+module.exports = [
+	{
+		name: 'indexMissingBlocks',
+		controller: indexMissingBlocks,
+		params: {},
+	},
+	{
+		name: 'indexNewBlock',
+		controller: indexNewBlock,
+		params: {
+			height: { optional: false, type: 'any' },
+		},
+	},
+	{
+		name: 'getMissingBlocksList',
+		controller: getMissingBlocksListByRange,
+		params: {
+			from: { optional: false, type: 'any' },
+			to: { optional: false, type: 'any' },
+		},
+	},
+	{
+		name: 'indexGenesisAccounts',
+		controller: indexGenesisAccounts,
+		params: {},
+	},
+	{
+		name: 'triggerAccountUpdates',
+		controller: triggerAccountUpdates,
+		params: {},
```

Comment:
Is this method responsible for indexing an account?
I think we are missing two methods here:
- `indexAccountByAddress`
- `indexAccountByPublicKey`

Reply:
We don't need to export any of these methods as the message queue will handle those internally.